App.js

```diff
@@ -68,19 +68,19 @@ function LeftHandNav({navigation}) {
   return (
     <View style={{flex: 1}}>
       <LHNHeader navigation={navigation} />
-      <Pressable style={{flexDirection: 'row', margin: 10, alignItems: 'center'}} onPress={() => navigation.push('Chat', {id: 1})}>
+      <Pressable style={{flexDirection: 'row', margin: 10, alignItems: 'center'}} onPress={() => navigation.navigate('Chat', {id: 1})}>
```

Comment:
The problem with the navigate is, let's say we navigated from chat 1 -> chat 2 -> chat 3, and if we click back to chat 1 then chat 2 and chat 3 will be removed from routes, and there is no way we can reach back to those 2 screens and because of that Marc was not agreeing to it.

Reply:
good point, I think push will be fine in the end right

Reply:
Yeah, we just need a check during the push operation to not push the page we are currently on.